### PR TITLE
Fix use_cache='' handling

### PR DESCRIPTION
**Pull request by olfway**
Do not use "/''" as cache directory when mounting with fstab

### Relevant Issue (if applicable)
 #1016 

### Details
When mounting from fstab it actually called with `use_cache=''` argument and it creates and uses `/''` directory as cache

### CODE REVIEW

**Comment by gaul** [comment]
Can the user just provide `use_cache=` instead?  I worry about inconsistently trying to handle quoting issues across all the parameters.

**Comment by olfway** [comment]
I've tried on centos 7
if I try to mount it with use_cache= parameter in fstab, I got error:
fuse: unknown option `use_cache'

fuse-2.9.2-11.el7.x86_64
s3fs-fuse-1.85-1.el7.x86_64

Also, in every man pages on the internet:
-o use_cache (default="" which means disabled)

**Comment by ggtakec** [comment]
@olfway I'm sorry for late reply.

I am hesitant about this proposal for two reasons.
- Common handling of optional parameters  
This is the same opinion as @gaul.  
There is no definition of quotes and double quotes in the current option specification method, and the specified all string is handled as an option parameter.  
In other words, if we supports like this PR code, it is rather than just using the use_cache option, I think that all options should handle parameters in the same way.
- How to make cache unspecified  
s3fs-fuse runs and disables the cache unless the use_cache option is specified.  
In other words, there is no need to specify use_cache="" option and its parameter.  
If we need the option to disable caching, we should add options such as "not_use_cache"(dedicated to disable).  
However, because the caching is disabled by default, this option is redundant and I think it will be confused users.

For the above reasons, we will not merge the contents of this PR.

If you need it for any reason(which we are not yet aware), please re-open it.
Thanks in advance for your kindness.
